doc/docs/Python_Tutorials/Basics.md

```diff
@@ -26,7 +26,7 @@ For our first example, let's examine the field pattern excited by a localized [C
 
 ### A Straight Waveguide
 
-The simulation script is in [examples/straight-waveguide.py](https://github.com/NanoComp/meep/blob/master/python/examples/straight-waveguide.py).
+The simulation script is in [examples/straight-waveguide.py](https://github.com/NanoComp/meep/blob/master/python/examples/straight-waveguide.py). The notebook is [examples/straight-waveguide.ipynb](https://github.com/NanoComp/meep/blob/master/python/examples/straight-waveguide.ipynb)
```

Comment:
Would be nicer to link via nbviewer, e.g. https://nbviewer.jupyter.org/github/NanoComp/meep/blob/master/python/examples/straight-waveguide.ipynb

Reply:
i.e. replace `https://github.com` with `https://nbviewer.jupyter.org/github` in the notebook links

Reply:
For example, see https://nbviewer.jupyter.org/github/mitmath/18335/blob/master/notes/Arnoldi.ipynb